app/ts/utils.ts

```diff
@@ -291,7 +291,7 @@ export const isValidEnsSubDomain = (subdomain: string): boolean => {
 	return ensRegex.test(subdomain)
 }
 
-const burnAddresses = ['0xdeaDDeADDEaDdeaDdEAddEADDEAdDeadDEADDEaD', '0x000000000000000000000000000000000000dEaD', '0x0000000000000000000000000000000000000000'] as const
+const burnAddresses = ['0xdeaDDeADDEaDdeaDdEAddEADDEAdDeadDEADDEaD', '0x000000000000000000000000000000000000dEaD', '0x0000000000000000000000000000000000000000', '0xdEaD000000000000000000000000000000000000', '0x0000000000000000000000000000000000000001'] as const
```

Comment:
I noticed that the `utils.ts` file primarily contains constant values. To improve clarity and organization, I suggest renaming `utils.ts` to `constants.ts`. 

Additionally, since there's already a `library/utilities.ts` file that contains utility functions, it might be worth considering moving functions from `utils.ts` to `library/utilities.ts` to maintain consistency.